cmd/askd/tests/form_submissions_test.go

```diff
@@ -0,0 +1,116 @@
+// Package tests implements users tests for the API layer.
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http/httptest"
+	"testing"
+
+	"gopkg.in/mgo.v2/bson"
+
+	"github.com/ardanlabs/kit/tests"
+	"github.com/gocarina/gocsv"
+)
+
+// relPrefix is the base name for everything.
+const relPrefix = "FTEST_"
+
+// TestExport tests the retrieval of a URL for a CSV file to download with the submissions.
+func TextExport(t *testing.T) {
+	tests.ResetLog()
+	defer tests.DisplayLog()
+
+	t.Log("Given the need download submissions in a CSV format.")
+	{
+		url := "/v1/form/57be0437e65ada0851000001/submission/export"
+		r := tests.NewRequest("GET", url, nil)
+		w := httptest.NewRecorder()
+
+		a.ServeHTTP(w, r)
+
+		t.Logf("\tWhen calling url : %s", url)
+		{
+			t.Log("\tWhen we user version 1.0 of the export endpoint.")
+			if w.Code != 200 {
```

Comment:
Should compare against `http.StatusOK` instead to keep things consistent throughout our codebase.